src/OpenTelemetry.Extensions.Enrichment/README.md

```diff
@@ -6,6 +6,87 @@
 Contains OpenTelemetry .NET SDK telemetry enrichment framework
 which is used for enrichment of traces.
 
+## Introduction
+
+Telemetry enrichment attaches various types of information to traces.
+Examples of the types of information that can be attached are environment
```

Comment:
environment, machine etc  are modelled as Resources in OpenTelemetry, so we should not encourage that to be modelled as Activity/Span.

Reply:
Rephrased. Actually, made this statement more generic. Hopefully this is fine